### PR TITLE
added some styling to contact page according to #49 

### PR DESCRIPTION
**Pull request by Ragzzy-R**
# Details
Seems like #49 was already 80% done. So I added some styling to it like fixing font-sizing, margin and colors. Also added a tiny animation for hover of social icon.

There seems to be an issue with fonts loading. seems like the fonts.scss was not imported to index.scss. added that too.

### CODE REVIEW

File: src/components/contact/assets/contact.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 section.contact {
   display: flex;
-  margin: 40px;
+  margin: 4rem;
```
Comment (emmawedekind) [inline src/components/contact/assets/contact.scss]: Please use a variable.

File: src/components/contact/assets/contact.scss [hunk 2]
```diff
@@ -57,4 +58,23 @@ section.contact {
   color: inherit;
   text-decoration: none;
   border-bottom: 1px solid $black;
+  cursor: pointer;
+}
+.slack_org:hover {
+  animation: slack-org-highlight 200ms ease-out forwards;
```
Comment (emmawedekind) [inline src/components/contact/assets/contact.scss]: You don't need `animation` for this. You can set the hover state and add a `transition` property to the selector.

File: src/components/contact/assets/contact.scss [hunk 3]
```diff
@@ -57,4 +58,23 @@ section.contact {
...
+}
+
+/*
+ Put animations specific to Component here.
+*/
+
+@keyframes slack-org-highlight {
```
Comment (emmawedekind) [inline src/components/contact/assets/contact.scss]: See above comment - you don't need a keyframe for this.

File: src/components/contact/assets/socialmedia.scss [hunk 4]
```diff
@@ -13,10 +13,40 @@
 .icon_wrapper {
   margin: 10px;
   padding: 15px;
-  border: 2px solid $green500;
+  border: 1px solid rgba($green500, 0.6);
   border-radius: 50%;
+  cursor: pointer;
+}
+.icon_wrapper:hover{
+  animation: social-icon-highlight 400ms ease-out forwards;
```
Comment (emmawedekind) [inline src/components/contact/assets/socialmedia.scss]: Again,  you can use `transition`. Please remove keyframes.

File: src/components/contact/assets/contact.scss (hunk 1 above)
Comment (Ragzzy-R) [reply src/components/contact/assets/contact.scss]: sure will do.

File: src/components/contact/assets/contact.scss (hunk 2 above)
Comment (Ragzzy-R) [reply src/components/contact/assets/contact.scss]: ohh ya totally forgot about transitions. will update ASAP.

**Comment by Ragzzy-R** [comment]
Hi @emmawedekind , Ive made the changes stated. please take a look :)

File: src/components/contact/assets/contact.scss [hunk 5]
```diff
@@ -26,11 +26,11 @@ section.contact {
     text-align: right;
     font-size: $font-xxl;
     color: $green500;
-    font-weight: 300;
+    font-weight: $font-weight-thin;
     text-transform: uppercase;
     border-bottom: 1px solid $gray;
     font-family: $font-fjalla-one;
-    margin-bottom: 0;
+    margin-bottom: $margin-none;
```
Comment (emmawedekind) [inline src/components/contact/assets/contact.scss]: I don't think we need a variable for margin...

File: src/components/contact/assets/contact.scss [hunk 6]
```diff
@@ -58,23 +58,11 @@ section.contact {
   color: inherit;
   text-decoration: none;
   border-bottom: 1px solid $black;
+  transition: background 300ms, color 300ms;
```
Comment (emmawedekind) [inline src/components/contact/assets/contact.scss]: What's the transition on these? Just linear?

Comment (emmawedekind) [reply src/components/contact/assets/contact.scss]: I would recommend ease-in or ease-out.

File: src/components/contact/assets/socialmedia.scss [hunk 7]
```diff
@@ -16,37 +16,19 @@
   border: 1px solid rgba($green500, 0.6);
   border-radius: 50%;
   cursor: pointer;
+  transition: background 400ms;
+  transition-timing-function: ease-out;
```
Comment (emmawedekind) [inline src/components/contact/assets/socialmedia.scss]: Add this to the transition property above.

File: src/components/contact/assets/socialmedia.scss [hunk 8]
```diff
@@ -16,37 +16,19 @@
...
-
-@keyframes social-icon-highlight-inverse {
-  0% {
-    color: $green500;
-  }
-  100% {
-    color: $color-white
-
-  }
+  transition: color 400ms;
```
Comment (emmawedekind) [inline src/components/contact/assets/socialmedia.scss]: Add this to the transition property above.

File: src/components/contact/assets/contact.scss (hunk 6 above)
Comment (Ragzzy-R) [reply src/components/contact/assets/contact.scss]: oh sorry! must have missed while typing.  doing it now

Comment (Ragzzy-R) [reply src/components/contact/assets/contact.scss]: done! :)

File: src/components/contact/assets/contact.scss [hunk 9]
```diff
@@ -59,6 +59,7 @@ section.contact {
   text-decoration: none;
   border-bottom: 1px solid $black;
   transition: background 300ms, color 300ms;
+  transition-timing-function: ease-out;
```
Comment (emmawedekind) [inline src/components/contact/assets/contact.scss]: Please remove this and add it to the `transition` property.

File: src/components/contact/assets/contact.scss (hunk 5 above)
Comment (Ragzzy-R) [reply src/components/contact/assets/contact.scss]: ohh i missed it. will push in next push if we have any more changes 👍

**Comment by Ragzzy-R** [comment]
done all the changes requested :) please let me know if anything else is needed

**Comment by Ragzzy-R** [comment]
Hi! please check the latest commits.

File: src/theme/index.scss [hunk 10]
```diff
@@ -1,6 +1,7 @@
 @import "base/variables";
 @import "base/elements";
 @import "base/mediaqueries";
+@import "base/fonts";
 //@import "base/fonts";
```
Comment (arthurbuhl) [inline src/theme/index.scss]: Please remove the unnecessary comment here. 

```scss
@import "base/fonts";
//@import "base/fonts";
```

File: src/components/contact/assets/contact.scss [hunk 11]
```diff
@@ -24,12 +24,13 @@ section.contact {
...
-    font-size: $font-xl;
-    color: $green300;
+    font-size: $font-xxl;
+    color: $green500;
+    font-weight: $font-weight-thin;
     text-transform: uppercase;
     border-bottom: 1px solid $gray;
     font-family: $font-fjalla-one;
-    margin-bottom: 0;
+    margin-bottom: $margin-none;
```
Comment (arthurbuhl) [inline src/components/contact/assets/contact.scss]:
```scss
margin-bottom: $margin-none;
```

`$margin-none` is still present here but has been removed from `_variables.scss`.

**Comment by Ragzzy-R** [comment]
I've updated the invalid declarations. pls take a look.

**Comment by emmawedekind** [comment]
I think it looks fine but @crysfel please check it out :)

**Pull request merged**